server/models/dtos/project_dto.py

```diff
@@ -89,6 +98,8 @@ class ProjectDTO(Model):
     priority_areas = BaseType(serialized_name='priorityAreas')
     last_updated = DateTimeType(serialized_name='lastUpdated')
     author = StringType()
+    task_creation_mode = StringType(required=True, serialized_name='taskCreationMode',
```

Comment:
Of course I can remove the "required" and "validators" if it helps merging this issue. I don't want this to be a blocker. We can then discuss it further. Or I can take a bit more time to analyze it.